### ISSUE TITLE
Ошибка при перетаскивании

### ISSUE THREAD

**Issue by losferwords** [Tuch/angular-dnd#36]
Привет, у меня создан объект такого плана:
```
<div class="gem"
                                     ng-repeat="gem in getGems(gemFilter) track by $index"
                                     ng-style="{'background-image': gem.image,
                                                    'background-color': gem.bgColor}"
                                     uib-tooltip-html="getGemTooltip(gem)"
                                     tooltip-placement="top"
                                     tooltip-append-to-body="true"
                                     tooltip-animation="false"

                                     dnd-draggable="true"
                                     dnd-draggable-opts = "{layer: 'layer1', helper : 'clone'}"
                                     dnd-on-dragstart = "inventoryDragStart(gem)"
                                     dnd-on-dragend = "inventoryDragEnd()">
                                </div>
```
При перетаскивании появляется ошибка: 
Uncaught TypeError: Cannot read property '$$phase' of null
drag @ angular-dnd.js:1913
Dnd.trigger @ angular-dnd.js:943
Manipulator.progress @ angular-dnd.js:1230
Mouse.mousemove @ angular-dnd.js:1341
(anonymous function) @ angular-dnd.js:83
n.event.dispatch @ jquery.js:4435
r.handle @ jquery.js:4121

Добавилась эта прелесть в 0.1.20, раньше всё было норм. Точно не знаю, чем это вызвано, но вот тут похожая вещь: https://github.com/angular-ui/ui-tinymce/issues/82

**Comment by Tuch** [comment]
@losferwords внес правки в новую версию 0.1.21

**Comment by Tuch** [comment]
А, кажется понял, у вас скоп где находится draggable элемент и есть rootScope.

**Comment by Tuch** [comment]
Вообщем я сделал ASAP решение, но все ксожелению глубже =( angular-dnd не отслеживает ситуацию, если scope уничтожается

**Comment by losferwords** [comment]
Версия angular 1.4.8, поставил на сервер версию bc1a86e, вроде как всё хорошо, спасибо за быстрый фикс) 

Да, ещё хотел указать на то, что bower не может определить версию вашего модуля. Если делать bower install angular-dnd-module --save, то в bower.json он добавит "*" зависимость, если делать bower install с зависимостью latest, то bower ставит последний коммит вида https://github.com/Tuch/angular-dnd.git#bc1a86e

**Comment by Tuch** [comment]
@losferwords, ok.

**Comment by Tuch** [comment]
@losferwords поправил тему с версией, но работает правда только для 0.1.22 сейчас, дальше будет обновляться при повышении версии.

**Comment by losferwords** [comment]
Да, всё отлично, большое спасибо) У тебя очень полезный модуль!

**Comment by Tuch** [comment]
@losferwords, спасибо, жаль в последнее время нет возможности его прокачивать =( Только вот такие хотфиксы =(
